packages/lesswrong/components/tagging/TagHoverPreview.tsx

```diff
@@ -53,7 +53,7 @@ const TagHoverPreview = ({href, targetLocation, innerHTML, classes, postCount=6,
     <Link
       className={showPostCount ? classes.linkWithoutDegreeSymbol : classes.link}
       to={linkTarget}
-      dangerouslySetInnerHTML={{__html: useTagName ? tag?.name : innerHTML}}
+      dangerouslySetInnerHTML={{__html: tagName ? tagName : innerHTML}}
```

Comment:
With the typing of `Link`, `dangerouslySetInnerHTML={{__html` was complaining about the value potentially being `null` (which the type system couldn't figure out was impossible from the control flow implied by the original `useTagName`).   This behaves the same way but doesn't cause the type system to throw a fit.

Reply:
Is there a reason you're doing this instead of `tagName || innerHTML`?

Reply:
Nope, also fixed the initial `tagName` assignment so I can do `tagName ?? innerHtml`, since otherwise `tagName` could sometimes be false (which would in fact require using `||` over `??`, but was just lazy on my part)